bench/src/main.rs

```diff
@@ -0,0 +1,65 @@
+//! Benchmark module loading performance.
+
+use bytes::Bytes;
+use vuln_reach::javascript::module::TarballModuleResolver;
+use vuln_reach::javascript::package::Package;
+use vuln_reach_master::javascript::module::TarballModuleResolver as UpstreamTarballModuleResolver;
+use vuln_reach_master::javascript::package::Package as UpstreamPackage;
+
+// Some useful test tarballs:
+//   https://registry.npmjs.org/core-js/-/core-js-3.30.2.tgz
+//   https://registry.npmjs.org/codemirror/-/codemirror-5.5.0.tgz
+//   https://registry.npmjs.org/mongoose/-/mongoose-7.2.1.tgz
+const PACKAGE_URI: &str = "https://registry.npmjs.org/mongoose/-/mongoose-7.2.1.tgz";
```

Comment:
I wouldn't do it as part of this PR, but we could make this into a list of packages and go through each of them in order to have a more comprehensive benchmark.